package.json

```diff
@@ -1,31 +1,32 @@
 {
-  "name": "metalsmith-clean-css",
-  "version": "5.0.1",
-  "description": "A Metalsmith plugin to minify CSS files",
-  "keywords": [
-    "metalsmith",
-    "plugin"
-  ],
+  "name": "metalsmith-clean-css-monorepo",
+  "private": true,
   "author": "Aymeric Beaumet <aymeric@beaumet.me>",
+  "extends": [
```

Comment:
I don't know about this?

Reply:
https://renovateapp.com/docs/ you'll have to activate it afterwards. It will auto update dependencies in your project.

Reply:
Nice!